.github/workflows/build_linux_pytorch_wheels.yml

```
@@ -12,7 +12,7 @@ on:
       python_version:
         required: true
         type: string
-      find_links:
+      s3_python_bucket:
```

```suggestion
      s3_bucket:
```

We can keep it shorter here as the whole workflow only operates on Python packages.